modules/docs/arrow-docs/docs/docs/generic/coproduct/README.md

```diff
@@ -178,16 +253,28 @@ fun handleApiResult(
                     { database.insertRegistration(it) }
             )
 }
+
+fun main() {
```

Comment:
you can use `//sampleStart` and `//sampleEnd` to remove noise from the examples such as imports main etc.